lib/diagnostic.js

```diff
@@ -14,9 +14,24 @@
 'use strict';
 
 const fs = require('fs'); // jshint ignore: line
-
+let inFile = filename;
 const sumLines = (filename, callback) => {
-  /* your response here */
+
+  fs.readFile(inFile, 'utf-8', (error, data) => {
```

Comment:
You passed a thing called `inFile` to readFile. But what is `inFile`? And why aren't you using the thing called `filename`? 🤔